docker-compose.yml

```diff
@@ -1,26 +1,52 @@
-# Apparently this attribute is obsolete? But NUS says we need this?
-#version: "3.9"
+version: "3.9"
```

Comment:
This attribute is obsolete, even if CS3219 says that this is needed in their documentation.